SPIE2016-9913-16/9913-16.tex

```diff
@@ -0,0 +1,303 @@
+\documentclass[]{spie}  %>>> use for US letter paper
+%\documentclass[a4paper]{spie}  %>>> use this instead for A4 paper
+%\documentclass[nocompress]{spie}  %>>> to avoid compression of citations
+
+\renewcommand{\baselinestretch}{1.0} % Change to 1.65 for double spacing
+\newcommand{\aap}{Astronomy \& Astrophysics}
+\newcommand{\procspie}{Proc.\ SPIE}
+\newcommand{\aspconf}{ASP Conf.\ Ser.}
+\newcommand{\qjras}{QJRAS}
+
+\newcommand{\ascl}[1]{\href{http://www.ascl.net/#1}{ascl:#1}}
+\newcommand{\arxiv}[1]{\href{http://arxiv.org/abs/#1}{arXiv:#1}}
+\newcommand{\doi}[1]{\href{http://dx.doi.org/#1}{doi:#1}}
+
+\usepackage{amsmath,amsfonts,amssymb}
+\usepackage{graphicx}
+\usepackage[colorlinks=true, allcolors=blue]{hyperref}
+
+\title{Investigating interoperability of the LSST Data Management software stack with Astropy}
+% Jenness, Bosch, Owen, Parejko, Sick, Swinbank, de Val-Boro et al
+\author[a]{Tim Jenness}
+\author[b]{James Bosch}
+\author[c]{Russell Owen}
+\author[c]{John Parejko}
+\author[a]{Jonathan Sick}
+\author[b]{John Swinbank}
+\author[b]{Miguel de Val-Borro}
+
+\affil[a]{LSST Project Management Office, Tucson, AZ, U.S.A.}
+\affil[b]{Princeton University, Princeton, NJ, U.S.A.}
+\affil[c]{University of Washington, Seattle, WA, U.S.A}
+
+\authorinfo{Further author information: (Send correspondence to T.J.)\\T.J.: E-mail: tjenness@lsst.org}
+
+% Option to view page numbers
+\pagestyle{empty} % change to \pagestyle{plain} for page numbers
+\setcounter{page}{301} % Set start page numbering at e.g. 301
+
+\begin{document}
+\maketitle
+
+\begin{abstract}
+  The Large Synoptic Survey Telescope (LSST) will be an 8.4\,m optical survey telescope sited in Chile and capable of imaging the entire sky twice a week.
+  The data rate of approximately 15\,TB per night and the requirements to both issue alerts on transient sources within 60 seconds of observing and also to create annual data releases means that automated data management systems and data processing pipelines are a key deliverable of the LSST construction project.
+  The LSST data management software has been in development since 2004 and, like other software developed in that era such as CASA, is based on a C++ core with a Python control layer.
```

Comment:
That is a possibility. Regardless, I'm happy to tweak the abstract on this a bit as people are happy with the version in the main text.